ansible/roles/schulcloud-server-provisioning/defaults/main.yml

```diff
@@ -0,0 +1,11 @@
+WITH_SCHULCONNEX_PROVISIONING: false
```

Comment:
The WITH_SCHULCONNEX_PROVISIONING setting for all must be add to https://github.com/hpi-schul-cloud/dof_app_deploy/blob/main/ansible/group_vars/all/with.yml